### PR TITLE
SPE-125 Aztec migration

### PR DESCRIPTION
**Pull request by couralex6**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: models/aztec/ethereum/aztec_v2_ethereum_rollupbridge_transfers.sql [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {{ config(
+    tags=['dunesql'],
     schema = 'aztec_v2_ethereum',
     alias = alias('rollupbridge_transfers'),
-    partition_by = ['evt_block_date'],
+    partition_by = ['evt_block_month'],
```
Comment (aalan3) [inline models/aztec/ethereum/aztec_v2_ethereum_rollupbridge_transfers.sql:5]: Can be dropped.

**Review by aalan3 — APPROVED** [review]

**Comment by couralex6** [comment]
@Hosuke do you know how to fix these failing tests by any chance?

File: models/aztec/ethereum/aztec_ethereum_schema.yml [hunk 2]
```diff
@@ -20,10 +20,6 @@ models:
             - evt_index
             - broad_txn_type
             - spec_txn_type
-            - to_protocol
-            - from_protocol
-            - bridge_address
-            - trace_address
```
Comment (Hosuke) [inline models/aztec/ethereum/aztec_ethereum_schema.yml:26]: I think `trace_address` here is important.

**Review by Hosuke — APPROVED** [review]
Good to go now.

**Comment by couralex6** [comment]
Thank you! @Hosuke

**Review by jeff-dude — APPROVED** [review]
thanks for helping @Hosuke!

**Pull request merged**